README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/JGCRI/hector.svg?branch=master)](https://travis-ci.org/JGCRI/hector)
+[![Build Status](https://travis-ci.org/JGCRI/hector.svg?branch=master)](https://travis-ci.org/JGCRI/hector) [![AppVeyor Build status](https://ci.appveyor.com/api/projects/status/4ld4ay2gyg2s8w6w?svg=true)](https://ci.appveyor.com/project/ashiklom/hector)
```

Comment:
NOTE: This badge needs to be updated to point to the JGCRI AppVeyor account. I'm not sure if I have all the right permissions to access it.

Reply:
I'll look into the appveyor config tomorrow.  Thanks for looking into this.